<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Server</class>
 <widget class="QMainWindow" name="Server">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Server</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="bobLabel">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>150</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: &quot;red&quot;</string>
    </property>
    <property name="text">
     <string>Bob</string>
    </property>
   </widget>
   <widget class="QLabel" name="eveLabel">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>340</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: &quot;red&quot;</string>
    </property>
    <property name="text">
     <string>Eve</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>591</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#on{
	background-image: url(:/on.png);
}

#off{
	background-image: url(:/off.png);
}</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="eveSpy">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>340</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Прослушивать канал</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="noiseGroup">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QSpinBox" name="noiseLevel">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>0</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="suffix">
      <string>%</string>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>5</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Уровень шума:</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="aliceLabel">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>140</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: &quot;red&quot;</string>
    </property>
    <property name="text">
     <string>Alice</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>31</width>
      <height>321</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="speedGroup">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>0</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Скорость передачи</string>
     </property>
    </widget>
    <widget class="QSlider" name="speedSlider">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>281</width>
       <height>17</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="singleStep">
      <number>50</number>
     </property>
     <property name="pageStep">
      <number>200</number>
     </property>
     <property name="value">
      <number>200</number>
     </property>
     <property name="sliderPosition">
      <number>200</number>
     </property>
     <property name="tracking">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>50</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="rejectGroup">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QSpinBox" name="photonProbability">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>0</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="suffix">
      <string>%</string>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>5</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="channelLength">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="suffix">
      <string> км</string>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>300</number>
     </property>
     <property name="singleStep">
      <number>25</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QRadioButton" name="photonProbabilityRadio">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>221</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Коэффициент передачи канала:</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="channelLengthRadio">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Длина канала:</string>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="rejectCheck">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>230</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Потери в канале</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="multiGroup">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QSpinBox" name="photonCount">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>0</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="value">
      <number>2</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Среднее число фотонов:</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="multiCheck">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Мультифотонность</string>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="noiseCheck">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Зашумление</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1020</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="loadFromFIle"/>
    <addaction name="saveToFile"/>
    <addaction name="separator"/>
    <addaction name="exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Сервер</string>
    </property>
    <addaction name="startStopAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="exit">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="loadFromFIle">
   <property name="text">
    <string>Загрузить настройки из файла</string>
   </property>
   <property name="toolTip">
    <string>Загрузить настройки из файла</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="saveToFile">
   <property name="text">
    <string>Сохранить настройки в файл</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="startStopAction">
   <property name="text">
    <string>Перезапустить сервер</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
